feature-libs/checkout/base/components/services/checkout-config.service.ts

```diff
@@ -20,7 +21,30 @@ export class CheckoutConfigService {
   private defaultDeliveryMode: Array<DeliveryModePreferences | string> =
     this.checkoutConfig.checkout?.defaultDeliveryMode || [];
 
-  constructor(private checkoutConfig: CheckoutConfig) {}
+  protected checkoutFlow =
+    this.checkoutFlowOrchestratorService?.getCheckoutFlow();
+
+  // TODO(NO_TICKET): make checkoutFlowOrchestratorService a required dependency
```

Comment:
Why NO_TICKET?

Reply:
If we use `inject` instead there is no need to create a ticket for the future removal.